api/RSSFeed.php

```diff
@@ -184,13 +184,17 @@ public function Description() {
 	}
 
 	/**
-	 * Output the feed to the browser
+	 * Output the feed to the browser.
+	 *
+	 * @param	SS_HTTPResponse|null	$response
+	 * @return	HTMLText
 	 */
-	public function outputToBrowser() {
+	public function outputToBrowser(SS_HTTPResponse $response = null) {
```

Comment:
Agreed, sadly. Also, IMHO, the `$response` parameter should be required so that we get explicit and consistent results rather than falling back on a singleton.